cmd/zetae2e/local/local.go

```diff
@@ -439,6 +439,9 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 				e2etests.TestSolanaDepositAndCallName,
 				e2etests.TestSolanaWithdrawAndCallName,
 				e2etests.TestSPLDepositAndCallName,
+				e2etests.TestSPLDepositAndCallRevertName,
```

Comment:
It's becoming a lot of tests that take a while to execute, if possible we should create a runner for Sol and one for SPL tests